src/sass/parse.js

```diff
@@ -4120,6 +4128,30 @@ function getS() {
 }
 
 /**
+ * Check if token is a space, newline, or a comment.
+ * @param {number} i Token's index number
+ * @return {number} Number of similar (space, newline, or comment) tokens
+ *      in a row starting with the given token.
+ */
+function checkSLC(i) {
```

Comment:
Can we rename the function to `checkMultilineSC` and update its description? As I understand, the main difference from existing `checkSC` is that it checks for tokens located on different lines, right? `checkSC` checks for newlines too, it's just the parser treats `\n` character as space at the end of the line rather than beginning of the next one.